src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -404,7 +404,7 @@ public void openPanel(String serviceProtocolUri, ContentManager contentManager,
 
     //noinspection CodeBlock2Expr
     ApplicationManager.getApplication().invokeLater(() -> {
-      new EmbeddedBrowser().openPanel(contentManager, tabName, url);
+      EmbeddedBrowser.getInstance().openPanel(contentManager, tabName, url);
```

Comment:
getInstance(project)
I think we'd want separate instances for each window.

Reply:
Hmm.. I can't do it easily, so hopefully only one project per window.